api/tests/test_game_state_server.py

```diff
@@ -35,26 +34,45 @@
 )
 VALID_UPDATE = {'action': 'create', 'data': asdict(VALID_TOKEN)}
 VALID_PING = Ping('ping_id', 'ping', 0, 0)
+_SENTINEL = object()
```

Comment:
These helper functions can be used in any of the tests, should probably be extracted into their own file